quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/OpenApiPropertyMappers.java

```diff
@@ -0,0 +1,34 @@
+package org.keycloak.quarkus.runtime.configuration.mappers;
+
+import org.keycloak.common.Profile;
+import org.keycloak.config.OpenApiOptions;
+
+import java.util.List;
+
+import static org.keycloak.quarkus.runtime.configuration.Configuration.isTrue;
+import static org.keycloak.quarkus.runtime.configuration.mappers.PropertyMapper.fromOption;
+
+public final class OpenApiPropertyMappers implements PropertyMapperGrouping {
+
+  @Override
+  public List<? extends PropertyMapper<?>> getPropertyMappers() {
+    return List.of(
+        fromOption(OpenApiOptions.OPENAPI_ENABLED)
```

Comment:
Wouldn't be better to use `fromFeature`? As when the openapi feature is enabled, the whole openapi is enabled as well instead of enabling feature and this property.

WDYT?

Can be done in a follow-up.

Reply:
ok, works for me